completions/completions_v2.py

```diff
@@ -79,10 +79,10 @@ def _run_complete():
     def should_run_completion_on_modified(self, view):
         current_location = view.sel()[0].end()
         is_disabled = is_tabnine_disabled(view)
-        in_json_end_line = is_json_end_line(view, current_location)
+        stop_completion_after_end_line = should_stop_completion_after_end_line(view, current_location)
         in_query_after_new_line = is_query_after_new_line(
             view, current_location)
-        return current_location - self._last_query_location >= COMPLEATIONS_REQUEST_TRESHOLD and not self._stop_completion and not is_disabled and not in_json_end_line and not in_query_after_new_line
```

Comment:
this is insane. Can we break it to multiple if statements that return a false?

Reply:
👍